acceptance/Gemfile

```diff
@@ -10,7 +10,7 @@ def location_for(place, fake_version = nil)
   end
 end
 
-gem "beaker", *location_for(ENV['BEAKER_VERSION'] || "~> 2.27")
+gem "beaker", *location_for(ENV['BEAKER_VERSION'] || "~> 2.32")
```

Comment:
Wanting to ensure we install at least 2.32.0 (but less than 3.0) since 2.32 is the first version that contains the correct `install_puppet_agent_dev_repo_on` implementation.